cli/azd/cmd/login.go

```diff
@@ -49,6 +65,9 @@ func loginCmdDesign(global *internal.GlobalCommandOptions) (*cobra.Command, *log
 	cmd := &cobra.Command{
 		Use:   "login",
 		Short: "Log in to Azure.",
```

Comment:
Can be follow-up issue: I think we should have a description that describes the behavior of what auth flow would be used, and how client-id, client-secret/ client-certificate will override user interactive auth flow.

Reply:
Added a `Long` help text that describes how this all works.